.github/workflows/ci.yml

```diff
@@ -4,14 +4,14 @@ on:
   pull_request:
   push:
     branches:
-      - master
+      - main
 
 jobs:
   test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: [ '2.4', '2.5', '2.6', '2.7', '3.0']
+        ruby: [ '2.6', '2.7', '3.0', '3.1' ]
```

Comment:
2.6 is EOL now.

Reply:
@damacus Drop the support of 2.6. Supporting only '2.7', '3.0', '3.1'